vulnerabilities/importers/__init__.py

```diff
@@ -95,7 +95,6 @@
         postgresql.PostgreSQLImporter,
         debian.DebianImporter,
         curl.CurlImporter,
-        epss.EPSSImporter,
```

Comment:
I know this is not ready for review yet, but we will not remove other importers until we stop the V1 support completely